cpp/src/Slice/PythonUtil.cpp

```diff
@@ -222,9 +222,44 @@ lookupKwd(const string& name)
     //
     static const string keywordList[] =
     {
-        "None", "and", "assert", "break", "class", "continue", "def", "del", "elif", "else", "except", "exec",
-        "finally", "for", "from", "global", "if", "import", "in", "is", "lambda", "not", "or", "pass",
-        "print", "raise", "return", "self", "try", "while", "yield"
+        "False",
+        "None",
+        "True",
+        "_",
+        "and",
+        "as",
+        "assert",
+        "async",
+        "await",
+        "break",
+        "case",
+        "class",
+        "continue",
+        "def",
+        "del",
+        "elif",
+        "else",
+        "except",
+        "finally",
+        "for",
+        "from",
+        "global",
+        "if",
+        "import",
+        "in",
+        "is",
+        "lambda",
+        "match",
+        "nonlocal",
+        "not",
+        "or",
+        "pass",
```

Comment:
Same for `print`, we cannot remove keywords in a patch release

```suggestion
        "pass",
        "print",
```